src/palace/manager/integration/license/opds/opds2/api.py

```diff
@@ -48,28 +74,155 @@ def __init__(self, _db: Session, collection: Collection):
             collection.integration_configuration.context.get(self.TOKEN_AUTH_CONFIG_KEY)
         )
 
-    @classmethod
+    def _get_saml_token_template_parameters(
+        self, patron: Patron
+    ) -> tuple[str | None, list[str] | None]:
+        """Get the SAML token arguments for a patron
+
+        :param patron: The patron to get parameters for
+
+        :return: A tuple containing the SAML entity ID and a list of person scoped affiliations,
+            or (None, None) if the patron has no SAML credentials, or they cannot be
+            extracted.
```

Comment:
This seems very specific to the T&F integration, but that might be because that's all we need to support right now (and all the SAML token properties we can support with the current `TemplateVariable` enum).

It might be helpful to add a comment that ties this to the SAML-related members of `TemplateVariable` and/or vice versa.